openpype/hosts/nuke/plugins/publish/collect_workfile.py

```diff
@@ -0,0 +1,40 @@
+import os
+import nuke
+import pyblish.api
+
+
+class CollectWorkfile(pyblish.api.InstancePlugin):
+    """Collect current script for publish."""
+
+    order = pyblish.api.CollectorOrder - 0.498
```

Comment:
I'm a bit worried that all the collectors in Nuke need such explicit ordering. Is that still needed considering how the pre-collection has changed with the new publisher?

Reply:
I think it is not necessary anymore on workfile collector.